src/View/Helper/I18nHelper.php

```diff
@@ -91,4 +101,85 @@ public function changeUrlLang($newLang) : string
 
         return $url;
     }
+
+    /**
+     * Translate object field
+     * Return translation (by response object and included data, field and language)
+     *
+     * @param array $object The object to translate
+     * @param array $included The included translations data
+     * @param string $attribute The attribute name
+     * @param string $lang The lang (2 chars string)
+     * @param bool $defaultNull Pass true when you want null as default, on missing translation
+     * @return string|null
+     */
+    public function field(array $object, array $included, string $attribute, string $lang, bool $defaultNull = false) : ?string
```

Comment:
what about this signature:
```php
public function field(array $object, string $attribute, string $lang, bool $defaultNull = false, array &$included = [])
{
}
```
if `included` is not passed we may look for an `included` array in view data as default
and common use in twig could be 
```
I18n.field(object, 'title', 'en)
```

Reply:
also `lang` may be omitted since is read from config

signature, may become:
```php
public function field(array $object, string $attribute, string $lang = '', bool $defaultNull = false, array &$included = [])
{
}
```
common use in twig will be: ```I18n.field(object, 'title')```